pkg/scheduler/DESIGN.md

```diff
@@ -248,35 +261,54 @@ This approach served two purposes:
 1. tenant fairness via a simple round-robin between all tenants with non-empty query request queues
 1. rudimentary tenant isolation via shuffle-shard assignment of noisy tenants to only a subset of queriers
 
-While this inter-tenant Quality-Of-Service approach has worked well,
-other QOS issues have arisen from the varying characteristics of Mimir's two "query components"
+While this inter-tenant Quality-of-Service approach has worked well,
+other QoS issues have arisen from the varying characteristics of Mimir's two "query components"
 utilized by the queriers to fetch TSDB data for executing queries: ingesters and store-gateways.
 
-### New Requirement: Queue Splitting by Query Component
+### New Requirement: Isolating Query Component Latency Effects
```

Comment:
updated this as Queue Splitting was not the requirement, fixing the observed problem was the requirement